src/python/deepgnn/graph_engine/snark/local.py

```diff
@@ -290,7 +290,7 @@ def edge_features(
         return self.graph.edge_features(
             np.copy(edges[:, 0]),
             np.copy(edges[:, 1]),
-            np.copy(edges[:, 2]),
+            np.array(edges[:, 2], dtype=np.int32),
```

Comment:
This will create a pointer to the original edges array I believe, maybe would be more inline to do np.copy(edges[:, 2]).to(np.int32)